egs/wsj/s5/steps/nnet3/xconfig_to_configs.py

```diff
@@ -159,6 +159,9 @@ def write_config_files(config_dir, all_layers):
     for layer in all_layers:
         try:
             pairs = layer.get_full_config()
+            if layer.descriptors.has_key('input') and 'input' in layer.descriptors['input']['final-string']:
```

Comment:
I know the prepended FixedAffineLayer is for LDA mat. However, i would like there is no limitation to implement other topos of neural networks without it. For saving computations for example.